markdoc.config.json

```diff
@@ -9,7 +9,8 @@
             "watch": true
         },
         "partials": {
-            "auth-security.md": "./src/partials/auth-security.md"
+            "auth-security.md": "./src/partials/auth-security.md",
+            "cli-disclaimer.md": "./src/partials/cli-disclaimer.md"
```

Comment:
why are we adding this? How has it been working without this? 🧐

Reply:
It's been working, but with lint errors. This fixes the errors

Reply:
I noticed the lint errors are still there after I reload the editor. So I've removed it.